03-attribute-operations.Rmd

```diff
@@ -787,7 +787,7 @@ These and other typical raster processing operations are part of the map algebra
 ```{block 03-attribute-operations-65, type='rmdnote'}
 Some function names clash between packages (e.g., a function with the name `extract()` exist in both **terra** and **tidyr** packages).
 This may lead to unexpected results when loading packages in a different order.
-In addition to not loading packages by referring to functions verbosely (e.g., `tidyr::extract()`), another way to prevent function names clashes is by unloading the offending package with `detach()`.
+In addition to loading packages by referring to functions verbosely (e.g., `tidyr::extract()`), another way to prevent function name clashes is by unloading the offending package with `detach()`.
```

Comment:
```suggestion
In addition to calling functions verbosely with their full namespace (e.g., `tidyr::extract()`) to avoid attaching packages with `library()`, another way to prevent function name clashes is by unloading the offending package with `detach()`.
```

Make more sense?